### PR TITLE
added logic to check if user and group should be managed by the modul…

### PR DESCRIPTION
**Pull request by cptcanuck**
…e or externally.  https://github.com/voxpupuli/puppet-kafka/issues/173

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -77,6 +77,8 @@
   $config_dir                       = $kafka::params::config_dir,
   Stdlib::Absolutepath $bin_dir     = $kafka::params::bin_dir,
   $log_dir                          = $kafka::params::log_dir,
+  $manage_group                     = $kafka::params::manage_group,
```
Comment (bastelfreak) [inline manifests/init.pp]: Can you add the `Boolean` datatypes here?

File: manifests/init.pp [hunk 2]
```diff
@@ -85,16 +87,22 @@
     }
   }
 
-  group { $group:
-    ensure => present,
-    gid    => $group_id,
+  validate_bool($manage_group)
```
Comment (bastelfreak) [inline manifests/init.pp]: no need for the validate_bool anymore.

**Comment by bastelfreak** [comment]
Hi @cptcanuck, thanks for the work. I added some inline comments. Can you add tests for your change? You can ping us 24/7 in #voxpupuli if you've any questions about puppet or tests.

File: manifests/init.pp (hunk 1 above)
Comment (cptcanuck) [inline manifests/init.pp]: made the requested change.  I'm not clever with git, do I need to make a new PR or does $magic happen to update this one?

Comment (cptcanuck) [inline manifests/init.pp]: huh the Boolean type isn't passing tests, nor syntax check on my side.  I've not strictly typed variables before, but using this document (http://www.example42.com/2015/09/09/puppet4-examples-data-types/) as a reference I seem to have the syntax correct.  Can you provide any direction for this?

**Comment by cptcanuck** [comment]
I'd be happy to learn how to do the testing stuff, but I've never done it before.  Any references or other tests on the project I can refer to?

**Comment by LionelCons** [comment]
This has been fixed recently. Please check that `master` works fine and close this PR.

**Pull request closed without merging**